src/data/resources/publications/01-fabric-network-infrastructure.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "FABRIC: A National-Scale Programmable Experimental Network Infrastructure"
+authors: "Ilya Baldin, Anita Nikolich, James Griffioen, Indermohan Inder S. Monga, Kuang-Ching Wang, Tom Lehman, Paul Ruth"
+link: https://ieeexplore.ieee.org/document/8972790
+doilink: https://doi.org/10.1109/MIC.2019.2958545
```

Comment:
this field may be redundant. if the DOI is known, then the DOI link is always `https://doi.org/<the-publication-DOI>`

Reply:
Good suggestion! I totally agree. I'll remove the doilink in data and modify the page accordingly.